# HG changeset patch
# User Rik <rik@octave.org>
# Date 1548199222 28800
#      Tue Jan 22 15:20:22 2019 -0800
# Branch stable
# Node ID de358b26dc882ee027dcf26435e8fa7ffbc8053d
# Parent  f6730533820e0357556626d994598e4cedde4c4a
trapz.m: Restore documentation comments lost in f6730533820e.

* trapz.m: Restore comments about what code in @example block is doing.

diff --git a/scripts/general/trapz.m b/scripts/general/trapz.m
--- a/scripts/general/trapz.m
+++ b/scripts/general/trapz.m
@@ -38,20 +38,23 @@
 ## actual spacing value (deltaX).  As an example, the integral of @math{x^3}
 ## over the range [0, 1] is @math{x^4/4} or 0.25.  The following code uses
 ## @code{trapz} to calculate the integral in three different ways.
 ##
 ## @example
 ## @group
 ## x = 0:0.1:1;
 ## y = x.^3;
+## ## No scaling
 ## q = trapz (y)
 ##   @result{} q = 2.5250
+## ## Approximation to integral by scaling
 ## q * 0.1
 ##   @result{} 0.25250
+## ## Same result by specifying @var{x}
 ## trapz (x, y)
 ##   @result{} 0.25250
 ## @end group
 ## @end example
 ##
 ## @seealso{cumtrapz}
 ## @end deftypefn
 
